src/frontend/src/sass/abstracts/_variables.scss

```diff
@@ -0,0 +1,12 @@
+// colors
+$color-primary-dark: #375e77;
```

Comment:
If this is all we're using SASS for, I'd say it's overkill.  Let's just use CSS variables and dump the extra complexity.

Reply:
I primarily added sass because it allows scoping and variable substitution which makes future style changes easier to implement. I agree though that if the complexity is not worth the returns then we can get rid of it

Reply:
I leave it up to those of you who will live in the CSS.  If we need it, keep it.  If we don't, it's one more thing people have to learn to get involved.

Reply:
@agarcia-caicedo what do you think? Would you like to keep the sass infrastructure?

Reply:
Like David said, it is an added level of complexity, and while I can see value from using sass, I don't think this project sufficiently benefits from its use

Reply:
I'll get rid of it then!